packages/cli/src/commands/generate/service/__tests__/__snapshots__/service.test.js.snap

```diff
@@ -189,6 +242,62 @@ describe('userProfiles', () => {
 "
 `;
 
+exports[`in typescript mode creates a multi word service test file with crud actions and only foreign as mandatory field 1`] = `
+"import {
+  transactions,
+  transaction,
+  createTransaction,
+  updateTransaction,
+  deleteTransaction,
+} from './transactions'
+import type { StandardScenario } from './transactions.scenarios'
+
+describe('transactions', () => {
+  scenario('returns all transactions', async (scenario: StandardScenario) => {
+    const result = await transactions()
+
+    expect(result.length).toEqual(Object.keys(scenario.transaction).length)
+  })
+
+  scenario(
+    'returns a single transaction',
+    async (scenario: StandardScenario) => {
+      const result = await transaction({ id: scenario.transaction.one.id })
+
+      expect(result).toEqual(scenario.transaction.one)
+    }
+  )
+
+  scenario('creates a transaction', async () => {
+    const result = await createTransaction({
+      input: { userId: 'scenario.transaction.two.userId' },
+    })
+
+    expect(result.userId).toEqual('scenario.transaction.two.userId')
+  })
+
+  scenario('updates a transaction', async (scenario: StandardScenario) => {
+    const original = await transaction({ id: scenario.transaction.one.id })
+    const result = await updateTransaction({
+      id: original.id,
+      input: { userId: 'scenario.transaction.two.userId' },
```

Comment:
Hmm...this should be outputting the actual variable value, but here it has quotes around it, which is definitely not correct! Was this happening even before this PR was added? It's possible this has been happening for a while, but I would have thought we'd have noticed by now—it would make all generated SDL/scaffold tests fail. Although the test and assertion on lines 265 and 267 are correct, no quotes.

Reply:
This case is only when the model doesn't have any editable non-foreign field, so foreign fields are the only option, and due to the bug (this MR is supposed to fix), I couldn't really generate tests before this PR. But this PR shouldn't impact this functionality, I think the issue already exists.